assets/javascripts/views/search/search.coffee

```diff
@@ -68,8 +68,10 @@ class app.views.Search extends app.View
     @value = @input.value
 
     if @value.length
+      app.document.showSidebar(false)
```

Comment:
This would be better done in the `app.views.Sidebar` class, which already toggles the search result view on/off:
[sidebar.coffee#L16-L18](https://github.com/Thibaut/devdocs/blob/4468f65dd86bead162ab5f4a39d1b853a0c55214/assets/javascripts/views/sidebar/sidebar.coffee#L16-L18)
[sidebar.coffee#L49-L55](https://github.com/Thibaut/devdocs/blob/4468f65dd86bead162ab5f4a39d1b853a0c55214/assets/javascripts/views/sidebar/sidebar.coffee#L49-L55)